CI.hs

```diff
@@ -48,7 +51,7 @@ data StackOptions = StackOptions
     , ghcOptions :: Maybe String -- If 'Just _', pass '--ghc-options="xxx"' to 'stack build' (for ghc verbose, try 'v3').
     } deriving (Show)
 
-data GhcFlavor = Ghc8101 | Ghc881 | Ghc882 | Ghc883 | DaGhc881 | GhcMaster String
+data GhcFlavor = Ghc8101 | Ghc881 | Ghc882 | Ghc883 | Da | GhcMaster String
```

Comment:
Ideally I'd like to see the DA specific parameters on the `Da` ctor as we do for the `GhcMaster` ctor. That `String` you see there is the commit SHA for GHC in the case of flavor `GhcMaster XXX`. Better we have one way of doing things so I'd rather see this mechanism extended if possible.